kubernetes-client-api/src/main/java/io/fabric8/kubernetes/client/utils/InputStreamPumper.java

```diff
@@ -87,6 +88,12 @@ public static void transferTo(InputStream in, Writable out) throws IOException {
     return CompletableFuture.runAsync(() -> {
       try {
         InputStreamPumper.transferTo(in, out);
+      } catch (InterruptedIOException e){
+        if (!Thread.currentThread().isInterrupted()) {
```

Comment:
Apologies for another comment, you need to remove this same `if-else` block from `catch (Exception e)` block

Reply:
In fact, I don't quite understand what you mean. You can modify it directly

Reply:
Is this correct?